packages/protocol/src/session/NodeSession.ts

```diff
@@ -47,9 +47,13 @@ export class NodeSession extends SecureSession {
     readonly #encryptKey: Bytes;
     readonly #attestationKey: Bytes;
     #caseAuthenticatedTags: CaseAuthenticatedTag[];
-    #isClosing = false;
     readonly supportsMRP = true;
     readonly type = SessionType.Unicast;
+    readonly #closedByPeer = AsyncObservableValue();
+    #isPeerLost = false;
+
+    // TODO - currently server specific; should move out or be integrated with client
+    readonly #subscriptions = new BasicSet<Subscription>();
```

Comment:
Yeahh formally subscriptions are "by peer" and should use "the current active session that peer has - or open a new one if none is there"